Client/mods/deathmatch/logic/luadefs/CLuaClientDefs.cpp

```diff
@@ -80,3 +82,20 @@ bool CLuaClientDefs::IsCapsLockEnabled()
 {
     return ((::GetKeyState(VK_CAPITAL) & 0x0001) != 0);
 }
+
+bool CLuaClientDefs::SetCursorColor(float r, float g, float b, float alpha) noexcept
+{
+    if (!g_pCore->IsMenuVisible())
+        g_pCore->GetGUI()->SetCursorColor(r, g, b, alpha);
+    else
+        g_pCore->GetGUI()->ResetCursorColor(r, g, b, alpha); // Force values to be updated
+
+    return true;
+}
+
+CLuaMultiReturn<float, float, float, float> CLuaClientDefs::GetCursorColor() noexcept
+{
+    float r, g, b, alpha;
+    g_pCore->GetGUI()->GetCursorColor(r, g, b, alpha);
+    return {r, g, b, alpha};
```

Comment:
Could you make the returned alpha argument in range 0-255, and not 0-1, like [getCursorAlpha](https://wiki.multitheftauto.com/wiki/SetCursorAlpha)

put `alpha * 255.f`

Reply:
For consistency.

Reply:
no need to call two functions getting alpha through one function a way easier